client/boot/index.js

```diff
@@ -211,7 +211,8 @@ function reduxStoreReady( reduxStore ) {
 		reduxStore.dispatch( setCurrentUserId( user.get().ID ) );
 		reduxStore.dispatch( setCurrentUserFlags( user.get().meta.data.flags.active_flags ) );
 
-		const participantInPushNotificationsAbTest = config.isEnabled('push-notifications-ab-test') && abtest('browserNotifications') === 'enabled';
+		const participantInPushNotificationsAbTest = config.isEnabled( 'push-notifications-ab-test' ) &&
```

Comment:
What we want to express is that

- if the push notification AB test is turned on, AND
- the user is eligible in EITHER of the active tests

then we consider the user participant.

Logical AND has higher precedence than Logical OR so expressing the intent I described, grouping is necessary here:

```
		const participantInPushNotificationsAbTest = config.isEnabled( 'push-notifications-ab-test' ) &&
			( abtest( 'browserNotifications' ) === 'enabled' || abtest( 'browserNotificationsPreferences' ) === 'enabled' );
```